src/assets/scripts/client/parsers/aircraftCommandParser/argumentValidators.js

```diff
@@ -168,6 +168,23 @@ export const fixValidator = (args = []) => {
     }
 };
 
+/**
+ * Returns true if the value is exactly a 3 digit decimal number
```

Comment:
I'd modify this slightly to: `Tests if value is exactly a 3 digit decimal number`